taxi/views.py

```diff
@@ -28,6 +31,22 @@ def index(request):
     return render(request, "taxi/index.html", context=context)
 
 
+@login_required
+def car_assign(request, pk: int):
+    user = request.user
+    car = Car.objects.get(id=pk)
+    car.drivers.add(user)
+    return redirect("taxi:car-detail", pk=pk)
+
+
+@login_required
+def car_unassign(request, pk: int):
+    user = request.user
+    car = Car.objects.get(id=pk)
+    car.drivers.remove(user)
+    return redirect("taxi:car-detail", pk=pk)
+
+
 class ManufacturerListView(LoginRequiredMixin, generic.ListView):
     model = Manufacturer
     context_object_name = "manufacturer_list"
```

Comment:
you don't need to add context_object_name if that's default value. You can read more here: [django class based views](https://docs.djangoproject.com/en/4.2/topics/class-based-views/generic-display/)